CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Okta Node SDK Changelog
 
 ## 6.0.0
+- [#291](https://github.com/okta/okta-sdk-nodejs/pull/291) Minimal supported Node version 12.22.0
```

Comment:
Wondering which change causes this node version upgrade?

Reply:
It came from `@typescript-eslint/eslint-plugin`: @typescript-eslint/eslint-plugin@5.3.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0"

Although its bump is not required for vuln fix, it seems to be fine to move node bar to 12.22 for major release.

Reply:
I feel it does not belong to any runtime deps, which means devs don't need to upgrade their node version to use version 6.0. Maybe still keep the old one?

Reply:
I am fine with keeping `^12.0` although this might be a bit inconvenient for devs checking out/building SDK locally (i.e. getting an error when installing modules).

Reply:
Reverted to `^12.0`